logger/logger.go

```diff
@@ -190,3 +201,17 @@ func (l *logger) printf(tag, msg string, args ...interface{}) {
 	l.logger.Output(2, s)
 	l.loggerMu.Unlock()
 }
+
+func (l *logger) getLogLevel(tag string) LogLevel {
```

Comment:
I think this method could be simplified to remove the guard clause and rely on `range` over an empty slice to fall through to the default return.

Reply:
You are right, done.